Member_Application.md

```diff
@@ -0,0 +1,7 @@
+# Member Application
+To be a member of OpenLab follow the simple steps below:
+
+* Submit an issue to [apply to join](https://github.com/theopenlab/joinin/issues/new) first, fill your basic information, the way you will support to OpenLab and so on
```

Comment:
```suggestion
* Open an issue to [apply to join](https://github.com/theopenlab/ocean/issues/new/choose) first, fill out the form completely with as much detail, for example the way you will support to OpenLab
```

Reply:
we still have to discuss the repo where to open the issue.